### PR TITLE
Remove robots references

### PR DESCRIPTION
**Pull request by sjledoux**
While the robots check was modified in PR #28, that change did not properly modify the description of the function, and it left some extraneous bits of code. This Pr fixes resolves those issues.

### CODE REVIEW

File: RwsCheck.py [hunk 1]
```diff
@@ -496,10 +492,9 @@ def find_robots_txt(self, check_sets):
                                         "'noindex' or 'none' tag in its header"
                                         )
                 except Exception as inst:
-                    if exception_retries not in str(inst):
-                        if exception_timeout not in str(inst):
-                            self.error_list.append(
-                                f"Unexpected error for service site: {service_site}; Received error: {inst}")
+                    if exception_timeout not in str(inst):
```
Comment (cfredric) [inline RwsCheck.py:495]: Should we remove the carveout for timeouts as well? I think we probably want to surface that error, too.

File: RwsCheck.py [hunk 2]
```diff
@@ -496,10 +491,8 @@ def find_robots_txt(self, check_sets):
                                         "'noindex' or 'none' tag in its header"
                                         )
                 except Exception as inst:
-                    if exception_retries not in str(inst):
-                        if exception_timeout not in str(inst):
-                            self.error_list.append(
-                                f"Unexpected error for service site: {service_site}; Received error: {inst}")
+                        self.error_list.append(
```
Comment (cfredric) [inline RwsCheck.py:494]: Looks like this is indented one level too far. (It'd be really nice if we had an autoformatter for the python in this repo.)

File: RwsCheck.py (hunk 1 above)
Comment (sjledoux) [reply RwsCheck.py]: Done.

**Review by cfredric — APPROVED** [review]

**Review by Parasos — COMMENTED** [review]
related_website_sets.JSON

**Review by Parasos — COMMENTED** [review]
RwsCheck.py

**Review by Parasos — COMMENTED** [review]
06fed53b50834363ca537be3e4231b285a3104f0

**Review by Parasos — COMMENTED** [review]
RwsCheck.py

**Review by Parasos — COMMENTED** [review]
check_sites.py

File: RwsCheck.py [hunk 3]
```diff
@@ -461,24 +461,19 @@ def find_invalid_alias_eSLDs(self, check_sets):
                         self.error_list.append(
                             f"The provided country code: {tld}, in: {site} is not a ICANN registered country code")
 
-    def find_robots_txt(self, check_sets):
+    def find_robots_tag(self, check_sets):
         """Checks service sites to see if they have a robots.txt subdomain.
```
Comment (Parasos) [inline RwsCheck.py:465]: tests/rws_tests.py

**Review by Parasos — COMMENTED** [review]
check_sites.py

**Review by yasin17496 — APPROVED** [review]

File: RwsCheck.py (hunk 2 above)
Comment (yasin17496) [reply RwsCheck.py]: - [ ] []()

Comment (yasin17496) [reply RwsCheck.py]: - [ ] []()[]()

**Pull request merged**